indico/modules/events/persons/templates/management/person_list.html

```diff
@@ -61,7 +61,14 @@
         </thead>
         <tbody>
             {% for person_data in persons -%}
-                <tr id="person-{{ person_data.person.id }}" class="i-table" data-person-roles="{{ person_data.roles | tojson | forceescape }}">
+                <tr id="person-{{ person_data.person.id }}" class="i-table"
+                    {% if person_data.roles.no_account %}
+                        data-no-account
+                    {% endif %}
+                    {% if person_data.person.invited_dt %}
+                        data-invited-already
+                    {% endif %}
+                    data-person-roles="{{ person_data.roles | tojson | forceescape }}" >
```

Comment:
move this up before the conditional ones. also, get rid of that extra space at the end ;)